frontend/src/static/js/v_authorship.js

```diff
@@ -97,16 +110,20 @@ window.vAuthorship = {
           out.lineCount = lineCnt;
           this.addLineCountToFileType(file.path, lineCnt, filesInfoObj);
 
-          const segments = this.splitSegments(file.lines);
-          out.segments = segments;
+          var segmentInfo = this.splitSegments(file.lines);
```

Comment:
Any reason why this is changed to `var`?

Reply:
have changed it back to ```const```